lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -196,6 +196,51 @@ def get_udids(app_id)
         response.body[:testerUdids] || []
       end
 
+      # Create testers
+      #
+      # args
+      #   project_number - Firebase project number
+      #   emails - An array of emails to be created as testers. A maximum of
+      #            1000 testers can be created at a time.
+      #
+      def add_testers(project_number, emails)
+        payload = { emails: emails }
+        connection.post(add_testers_url(project_number), payload.to_json) do |request|
+          request.headers[AUTHORIZATION] = "Bearer " + @auth_token
+          request.headers[CONTENT_TYPE] = APPLICATION_JSON
+        end
+      rescue Faraday::BadRequestError
+        UI.user_error!(ErrorMessage::INVALID_EMAIL_ADDRESS)
+      rescue Faraday::ResourceNotFound
+        UI.user_error!(ErrorMessage::INVALID_PROJECT)
+      rescue Faraday::ClientError => e
+        if e.response_status == 429
```

Comment:
Does the FE actually do tester limit checking?

Reply:
Sorry, I meant BE*

Reply:
Yep we check for tester limits on the backend